+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Tue Aug  8 00:39:53 2023 - INFO  - Start environmental preparation.
+ service=system-update-cleanup.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-08-08 00:39:53'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Tue Aug  8 00:39:53 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Tue Aug  8 00:39:54 2023 - INFO  - Start testing...
+ P_SSH_CMD --node 2 --cmd 'touch /system-update;systemctl restart system-update-cleanup.service'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'touch /system-update;systemctl restart system-update-cleanup.service'
Tue Aug  8 00:39:56 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ REMOTE_REBOOT_WAIT 2 60
+ node=2
+ waittime=60
+ '[' 60x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 60
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
    ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
TypeError: can only concatenate str (not "NoneType") to str
+ P_SSH_CMD --node 2 --cmd 'test ! -f /system-update'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'test ! -f /system-update'
Tue Aug  8 00:40:00 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'system-update-cleanup.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='system-update-cleanup.service restart failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'system-update-cleanup.service restart failed'
+ LOG_ERROR 'system-update-cleanup.service restart failed'
+ message='system-update-cleanup.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'system-update-cleanup.service restart failed'
Tue Aug  8 00:40:00 2023 - ERROR - system-update-cleanup.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 34'
+ message='oe_test_service_system-update-cleanup.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 34'
Tue Aug  8 00:40:01 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ P_SSH_CMD --node 2 --cmd 'systemctl stop system-update-cleanup.service'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'systemctl stop system-update-cleanup.service'
Tue Aug  8 00:40:03 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'system-update-cleanup.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='system-update-cleanup.service stop failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'system-update-cleanup.service stop failed'
+ LOG_ERROR 'system-update-cleanup.service stop failed'
+ message='system-update-cleanup.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'system-update-cleanup.service stop failed'
Tue Aug  8 00:40:04 2023 - ERROR - system-update-cleanup.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 36'
+ message='oe_test_service_system-update-cleanup.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 36'
Tue Aug  8 00:40:04 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ P_SSH_CMD --node 2 --cmd 'systemctl status system-update-cleanup.service | grep '\''Active: inactive'\'''
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'systemctl status system-update-cleanup.service | grep '\''Active: inactive'\'''
Tue Aug  8 00:40:12 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'system-update-cleanup.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='system-update-cleanup.service stop failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'system-update-cleanup.service stop failed'
+ LOG_ERROR 'system-update-cleanup.service stop failed'
+ message='system-update-cleanup.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'system-update-cleanup.service stop failed'
Tue Aug  8 00:40:12 2023 - ERROR - system-update-cleanup.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 39'
+ message='oe_test_service_system-update-cleanup.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 39'
Tue Aug  8 00:40:13 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ P_SSH_CMD --node 2 --cmd 'systemctl is-enabled system-update-cleanup.service | grep '\''static'\'''
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'systemctl is-enabled system-update-cleanup.service | grep '\''static'\'''
Tue Aug  8 00:40:15 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'The unit files is not static'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='The unit files is not static'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'The unit files is not static'
+ LOG_ERROR 'The unit files is not static'
+ message='The unit files is not static'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'The unit files is not static'
Tue Aug  8 00:40:15 2023 - ERROR - The unit files is not static
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 41'
+ message='oe_test_service_system-update-cleanup.sh line 41'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 41'
Tue Aug  8 00:40:16 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 41
+ '[' 0 -eq 1 ']'
+ return 0
+ P_SSH_CMD --node 2 --cmd 'touch /system-update;systemctl start system-update-cleanup.service'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'touch /system-update;systemctl start system-update-cleanup.service'
Tue Aug  8 00:40:18 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ REMOTE_REBOOT_WAIT 2 60
+ node=2
+ waittime=60
+ '[' 60x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 60
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
    ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
TypeError: can only concatenate str (not "NoneType") to str
+ P_SSH_CMD --node 2 --cmd 'test ! -f /system-update'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'test ! -f /system-update'
Tue Aug  8 00:40:22 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'system-update-cleanup.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='system-update-cleanup.service start failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'system-update-cleanup.service start failed'
+ LOG_ERROR 'system-update-cleanup.service start failed'
+ message='system-update-cleanup.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'system-update-cleanup.service start failed'
Tue Aug  8 00:40:22 2023 - ERROR - system-update-cleanup.service start failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 45'
+ message='oe_test_service_system-update-cleanup.sh line 45'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 45'
Tue Aug  8 00:40:23 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 45
+ '[' 0 -eq 1 ']'
+ return 0
+ P_SSH_CMD --node 2 --cmd 'test 0 -eq $(journalctl --since '\''2023-08-08 00:39:53'\'' -u system-update-cleanup.service | grep -i -c '\''fail\|error'\'')'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'test 0 -eq $(journalctl --since '\''2023-08-08 00:39:53'\'' -u system-update-cleanup.service | grep -i -c '\''fail\|error'\'')'
Tue Aug  8 00:40:25 2023 - ERROR - You need to check the environment configuration file to see if this node information exists.
+ CHECK_RESULT 1 0 0 'There is an error message for the log of system-update-cleanup.service'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='There is an error message for the log of system-update-cleanup.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'There is an error message for the log of system-update-cleanup.service'
+ LOG_ERROR 'There is an error message for the log of system-update-cleanup.service'
+ message='There is an error message for the log of system-update-cleanup.service'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of system-update-cleanup.service'
Tue Aug  8 00:40:25 2023 - ERROR - There is an error message for the log of system-update-cleanup.service
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_system-update-cleanup.sh line 47'
+ message='oe_test_service_system-update-cleanup.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_system-update-cleanup.sh line 47'
Tue Aug  8 00:40:26 2023 - ERROR - oe_test_service_system-update-cleanup.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Tue Aug  8 00:40:26 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Aug  8 00:40:27 2023 - ERROR - failed to execute the case.
+ exit 6
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_service_system-update-cleanup.sh post_test'
+ message='oe_test_service_system-update-cleanup.sh post_test'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'oe_test_service_system-update-cleanup.sh post_test'
Tue Aug  8 00:40:27 2023 - INFO  - oe_test_service_system-update-cleanup.sh post_test
